src/components/Filters/FiltersList.vue

```diff
@@ -99,10 +99,6 @@ export default {
   display: inline-block;
   padding: 1.5rem 1rem 0 1.5rem;
 }
-.scroll-y {
-  overflow-y: scroll;
-  height: calc(100vh - 84px);
```

Comment:
A comment explaining the calculation behind `84px` would be nice for future reference.

Reply:
these are lines that were removed 😄 othwise i'd agree!